test/lit.cfg

```diff
@@ -1474,7 +1470,7 @@ elif run_os == 'linux-androideabi' or run_os == 'linux-android':
         '-sdk', config.variant_sdk, '-Xclang-linker',
         '--target={}{}'.format(config.variant_triple, config.android_api_level),
         '-tools-directory', tools_directory,
```

Comment:
Here we are keeping the `-tools-directory` (which I think it is the same as `-B`). This will mean that during compilation one set of tools will be used, while the tests will use a different one.

I am not saying this is wrong, but I want to make sure we are making every change, and there's a reason.

(There's another case below)

Reply:
Perhaps, but if so that means the trunk Android CI always worked this way till last May, ie most of the time. I'm all for regularizing it now though, see below.